tests/ui/process/process-sigpipe.rs

```diff
@@ -35,7 +35,7 @@ fn main() {
     assert!(output.stderr.len() == 0);
 }
 
-#[cfg(not(unix))]
+#[cfg(any(not(unix), target_os = "emscripten"))]
 fn main() {
     // Not worried about signal masks on other platforms
 }
```

Comment:
Suggestion: this should use both:

1. `//@ only-unix`, and
2. `//@ ignore-emscripten`

So they get properly registered as ignored instead of stubbing out what's being tested. Please also leave a remark of why the `only-*` and `ignore-emscripten` are needed.